packages/tc-api-db-manager/index.js

```diff
@@ -4,7 +4,8 @@ const logger = require('@financial-times/n-logger').default;
 const schema = require('@financial-times/tc-schema-sdk');
 const dbConnection = require('./db-connection');
 
-const exclusion = (arr1, arr2) => arr1.filter(val => !arr2.includes(val));
+const exclusion = (arr1, arr2) =>
```

Comment:
I'm surprised this is needed. think there might be a mistake somewhere in flattening the nested lists of indexes/constraints into a single array.

We're on node12 now, so you could use [flatMap](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap) to simplify the horrible code from line 44 onwards